src/apps/relay/prom_server.c

```diff
@@ -41,7 +45,15 @@ void start_prometheus_server(void) {
     nlabels++;
   }
 
-  // Create traffic counter metrics
+  // Create STUN counters
+  stun_binding_request = prom_collector_registry_must_register_metric(
+      prom_counter_new("turn_traffic_rcvp", "Incoming STUN Binding requests", 0, NULL));
```

Comment:
the name of the metrics cannot be the same for the 3 metrics, right?

Reply:
I'd actually use the same name but different labels. That should make the dev's live easier when drawing a dashboard and reading the metrics.

I'd suggest:
`const char* label[] = { "type" };
stun_binding = prom_collector_registry_must_register_metric(
      prom_counter_new("turn_traffic_stun", "STUN Binding requests", 1, label));`

usage:
`const char* type[] = { "error" };
prom_counter_add(stun_binding_request, 1, type);`

This should result in way nicer (and less!) metrics on the prometheus DB and thus give performance.

Disclaimer:
Probably 
`stun_binding = prom_collector_registry_must_register_metric(
      prom_counter_new("turn_traffic_stun", "STUN Binding requests", 1, (const char* []) { "type" }));`
  
should also work BUT my C knowledge is quite rusty.

Reply:
Thought a bit about it.

This:
`stun_binding = prom_collector_registry_must_register_metric( prom_counter_new("turn_traffic_stun", "STUN Binding requests", 1, (const char* []) { "type" }));`

Is also broken. Do not use that! It's again an "taking address of temporary array" error when compiling with C++ compiler flags.